### PR TITLE
Bookmarking an Item

### PR DESCRIPTION
**Pull request by sloecke**
Created the backend for handling adding and removing bookmarks. Edited the item show view to have a new bookmark button

### CODE REVIEW

**Review by SamNicklez — APPROVED** [review]
looks good

File: app/models/item.rb
Comment (LukeFarmer20) [inline app/models/item.rb:1]: What's the purpose of the :bookmarked_by?

Comment (sloecke) [reply app/models/item.rb:1]: It's just a way to get the users that have book marked an item from the item. I didn't end up using it anywhere yet so we could take it out

Comment (LukeFarmer20) [reply app/models/item.rb:1]: Ok, sounds good to me

**Review by LukeFarmer20 — APPROVED** [review]
Good work

**Review by ColeArduser — APPROVED** [review]

**Pull request merged**